LeoProjectGaming/src/main/java/com/work/LeoProjectGaming/entity/Bet.java

```diff
@@ -0,0 +1,43 @@
+package com.work.LeoProjectGaming.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import java.util.Collection;
+import java.util.Date;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@ToString
+@Entity
+public class Bet {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "Id")
+    private int id;
+
+    @Column(name = "Playerid")
+    private int playerId;
+
+    @Column(name = "Betamount")
+    private double betAmount;
+
+    @Column(name = "Betwinamount")
+    private double betWinAmount;
+
+    @Column(name = "Betloseamount")
+    private double betLoseAmount;
```

Comment:
I dont think it makes sense with betLose. 
the logic is like this - I bet 10 and win 100, bet amount 10, bet win 100,
I bet 10 and loose - bet amount 10, bet win - 0